cli/src/pcluster/models/cluster.py

```diff
@@ -654,4 +654,7 @@ def _add_version_tag(self):
 
     def _get_cfn_tags(self):
         """Return tag list in the format expected by CFN."""
-        return [{"Key": tag.key, "Value": tag.value} for tag in self.config.tags]
+        return [{"Key": tag.key, "Value": tag.value} for tag in self.config.tags] + [
+            {"Key": PCLUSTER_S3_BUCKET_TAG, "Value": self.bucket.name},
+            {"Key": PCLUSTER_S3_CLUSTER_DIR_TAG, "Value": self.bucket.artifact_directory},
```

Comment:
Could you add more description in the commit message? Why do we need this tag?

Reply:
Let's talk about it